modules/prometheus-database-storage-requirements.adoc

```diff
@@ -3,20 +3,26 @@
 // * scalability_and_performance/recommended-performance-scale-practices/recommended-infrastructure-practices.adoc
 // * installing-byoh/installing-existing-hosts.adoc
 
+:_mod-docs-content-type: REFERENCE
 [id="prometheus-database-storage-requirements_{context}"]
 = Prometheus database storage requirements
 
-Red Hat performed various tests for different scale sizes.
+Red{nbsp}Hat performed various tests for different scale sizes.
 
 [NOTE]
 ====
-The Prometheus storage requirements below are not prescriptive and should be used as a reference. Higher resource consumption might be observed in your cluster depending on workload activity and resource density, including the number of pods, containers, routes, or other resources exposing metrics collected by Prometheus.
+The following Prometheus storage requirements are not prescriptive and should be used as a reference. Higher resource consumption might be observed in your cluster depending on workload activity and resource density, including the number of pods, containers, routes, or other resources exposing metrics collected by Prometheus.
+====
+
+[NOTE]
+====
+You can configure the size-based data retention policy to suit your storage requirements.
```

Comment:
Up to you, but one alternative you can do to avoid the separate notes stacked on top of each other is to use a bullet list within one note:
```suggestion
* The following Prometheus storage requirements are not prescriptive and should be used as a reference. Higher resource consumption might be observed in your cluster depending on workload activity and resource density, including the number of pods, containers, routes, or other resources exposing metrics collected by Prometheus.

* You can configure the size-based data retention policy to suit your storage requirements.
```

The IMPORTANT admonition at the bottom of this Prereqs section is a live example of what that would look like: https://docs.openshift.com/container-platform/4.15/updating/updating_a_cluster/updating-cluster-cli.html#prerequisites

Reply:
Hello @skopacz1 , thank you for the suggestion! I never know what looks better and I have not seen any guideline about this. I like your idea, thanks!